### ISSUE TITLE
TouchableOpacity onPress event being called twice

### ISSUE THREAD

**Issue by jnvbui** [microsoft/react-native-windows#2897]
### Environment

1. `react-native -v`:
react-native-cli: 2.0.1
react-native: 0.59.0-microsoft.17

2. `npm ls rnpm-plugin-windows`:
-- rnpm-plugin-windows@0.2.11

3. `npm ls react-native-windows`:
-- react-native-windows@0.59.0-vnext.109

4. `node -v`:
v10.16.0

5. `npm -v`:
6.10.0

Then, specify:

- Target Platform Version(s):  10.0.18362

- Target Device(s): Xbox

- Visual Studio Version: 2019

- Build Configuration: Debug

### Steps to Reproduce
1. Create an TouchableOpacity component
2. Set onPress event handler
3. After clicking on it, notice that the event gets called twice

### Expected Behavior
onPress event gets called once when the TouchableOpacity component is pressed.

### Actual Behavior
onPress event gets called twice when the TouchableOpacity component is pressed.

### What could be causing the issue
I have an onPress event for a TouchableOpacity that pushes to the memoryHistory stack and navigates to another page. However, I noticed that the history stack pushes twice every time.
I found that this is caused by the `onKeyDown` and `onKeyUp` handlers in TouchableOpacity.uwp.js.

`_onKeyUp` works as expected but `_onKeyDown` calls `touchableHandleActivePressIn()` which has the following definition:
```
touchableHandleActivePressIn: function(e: PressEvent) {
    if (e.dispatchConfig.registrationName === 'onResponderGrant') {
      this._opacityActive(0);
    } else {
      this._opacityActive(150);
    }
    this.props.onPressIn && this.props.onPressIn(e);
  }
```

Since `_onKeyDown` calls `touchableHandleActivePressIn` without an argument, the first if statement would throw an exception.  As a result, onPress gets called again for some reason, possibly due to the exception handling somewhere in the code.

**Comment by jnvbui** [comment]
@harinikmsft 

I added a check to make sure `e` exists before calling `e.dispatchConfig.registrationName === 'onResponderGrant'` in the `if` statement.

However I noticed that our onPress event is still being called twice due to `this.touchableHandlePress()` being called on KeyUp and `this.touchableHandleActivePressIn()` being called on KeyDown, which both call `this.props.onPress(e)`.

I was just wondering if it is possible to make `onKeyUp` and `onKeyDown` only call the onPress event once?
